.circleci/config.yml

```diff
@@ -862,12 +862,44 @@ jobs:
       - run:
           name: Run Tests
           command: yarn --cwd=packages/phone-number-privacy test
-
+  
+  certora-test:
+    working_directory: ~/app
+    docker:
+      - image: circleci/openjdk:11-jdk
+    steps:
+      - attach_workspace:
+          at: ~/app
+      - run:
```

Comment:
Per https://github.com/celo-org/celo-monorepo/pull/2721/files#r423128751, can we add the following lines before installing certora dependencies:

```
      - run:
          name: Check if the test should run
          command: |
            ./scripts/ci_check_if_test_should_run_v2.sh @celo/protocol
```

IIUC this should prevent the test from running if nothing has changed in the protocol package or its dependencies

Reply:
Added when resolving conflicts